### DISCUSSION TITLE
Seeking help on doc/information about the compiled database from GCBM.Belize

### DISCUSSION THREAD

**Discussion by ankitaS11** [moja-global/moja_global_docs#195] [Q&A]
Hi,

I’m trying to generate graphs explaining NPP, NEP etc. from the GCBM.Belize run, and am unable to find the relevant table names and column names for the same. Like, I figured out that there is `v_flux_indicators` table, but again, not too sure what `flux_tc` column is?

Is there any documentation or a script that has done this before? By this, I mean, exploring the compiled database, and/or plotting some metrics.

cc: @aornugent

**Comment by aornugent** [comment]
Hi @ankitaS11!

I haven't seen this documented (but it must be somewhere) - the exported indicators are described in `modules_outputs.json`. 

For example here's how to add [NPP](https://github.com/moja-global/FLINT.Cloud/blob/b6c0f6008f2001946e63f4e0cd372c8b91fc8c7a/local/rest_api_gcbm/templates/modules_output.json#L107)  - it's the flux of carbon from the atmosphere into living vegetative biomass. `flux_tc` means tonnes of carbon.

**Comment by aornugent** [comment]
Here's a diagram of the fluxes described in the CBM-CFS3 Science modules (which are built in to the GCBM)

HW stands for hardwood and SW stands for softwood. The decomposition into specific gasses (CO, CO2, CH4) is not shown, but is also available. This is important because each gas has a different global warming potential.

![Screenshot_20220611-231936](https://user-images.githubusercontent.com/20387761/173189707-c2a07923-d240-4a07-bdab-8ca8a148ad61.png)
